### PR TITLE
small screens

### PR DESCRIPTION
**Pull request by SharonV33**
fix layout of donut chart section on tiny screens

### CODE REVIEW

**Review by johancvb — APPROVED** [review]

File: src/components/atoms/Dropdown.svelte [hunk 1]
```diff
@@ -30,6 +30,15 @@
...
   }
+
+  @media (max-width: 350px) {
+    section {
+      margin-bottom: 0.5em;
+      select {
+        width: 100%;
+      }
+    }
+  }
```
Comment (veerleprins) [inline src/components/atoms/Dropdown.svelte]: Wouldn't it be smarter to take this media query out and just grab it from 320px with styling?

File: src/components/molecules/DonutDropdown.svelte [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+  @media (max-width: 350px) {
+    div {
+      flex-direction: column;
+
+      div {
+        width: 100%;
+      }
+    }
+  }
+
```
Comment (veerleprins) [inline src/components/molecules/DonutDropdown.svelte]: Same here ? (See previous note)

File: src/components/atoms/Legend.svelte [hunk 3]
```diff
@@ -42,6 +42,12 @@
       }
     }
   }
+
+  @media (max-width: 350px) {
+    ul {
+      display: block;
+    }
+  }
```
Comment (veerleprins) [inline src/components/atoms/Legend.svelte]: Same here ? (See previous note)

File: src/components/molecules/DonutDropdown.svelte (hunk 2 above)
Comment (SharonV33) [reply src/components/molecules/DonutDropdown.svelte]: good tip, implemented

**Review by veerleprins — APPROVED** [review]
Yes looks good 👍🏼

**Pull request merged**